pygmt/datasets/earth_relief.py

```diff
@@ -38,6 +38,12 @@ def load_earth_relief(resolution="01d", region=None, registration=None):
         ``'03m'``, ``'02m'``, ``'01m'``, ``'30s'``, ``'15s'``, ``'03s'``,
         or ``'01s'``.
 
+    use_srtm : bool or str
+        By default, the land-only SRTM tiles from NASA are used to generate the
+        ``'03s'`` and ``'01s'`` grids, along with the missing ocean values
+        filled by up-sampling the relevant SRTM15+V2.1 tiles. If True, will
+        only load the original land-only SRTM tiles.
+
```

Comment:
Please move the description of the `use_srtm` to the end of the parameter list.

Reply:
Done in 3b4d482.